mathematica/ieee754_floating_point_evaluation.wl

```diff
@@ -35,17 +39,17 @@ Options[IEEEEvaluate]={UseFMA->True};
 IEEEEvaluate[x:(_Plus|_Times|_Power|_?NumberQ),OptionsPattern[]]:=
 Block[
 {Plus,Times,ev},
```

Comment:
Do you still need Plus and Times in the block?

Reply:
Curiously I do.  I think it just declare local versions of these functors.  If I remove them I get the "normal" ones which are `Flat` and everything falls apart.